src/main/resources/add-ons-to-index.json

```diff
@@ -152,6 +158,9 @@
       "uid": "org.openmrs.module.appframework",
       "type": "OMOD",
       "name": "App Framework Module",
+      "tags": [
+        "Framework", "App"
```

Comment:
Taking part of the module name and including it as a tag isn't really helpful, kindly try to add meaningful tags.